webapp-django/crashstats/crashstats/templatetags/jinja_helpers.py

```diff
@@ -187,7 +187,12 @@ def bugzilla_submit_url(report, parsed_dump, crashing_thread, bug_product):
         "comment": comment,
     }
 
-    # some special keys have to be truncated to make Bugzilla happy
+    # If this crash report has DOMFissionEnabled=1, then we prepend "[Fission]"
+    # to the summary. This is probably temporary. Bug #1659175.
+    if raw_crash.get("DOMFissionEnabled"):
```

Comment:
This annotation is either there (if fission is enabled) or not there at all (if fission isn't enabled or fission isn't a thing in this product).